frontend/sw360-portlet/src/main/webapp/html/admin/todos/add.jsp

```diff
@@ -0,0 +1,101 @@
+<%--
+  ~ Copyright Siemens AG, 2013-2017. Part of the SW360 Portal Project.
+  ~
+  ~ SPDX-License-Identifier: EPL-1.0
+  ~
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/legal/epl-v10.html
+  --%>
+<%@ page import="org.eclipse.sw360.portal.common.PortalConstants" %>
+<%@include file="/html/init.jsp"%>
+<%-- the following is needed by liferay to display error messages--%>
+<%@include file="/html/utils/includes/errorKeyToMessage.jspf"%>
+<portlet:defineObjects />
+<liferay-theme:defineObjects />
+
+<%@ page import="javax.portlet.PortletRequest" %>
+<%@ page import="com.liferay.portlet.PortletURLFactoryUtil" %>
+<%@ page import="org.eclipse.sw360.datahandler.thrift.licenses.Todo" %>
+
+
+<jsp:useBean id="todo" class="org.eclipse.sw360.datahandler.thrift.licenses.Todo" scope="request" />
+
+
+
+<portlet:actionURL var="addURL" name="addTodo">
+</portlet:actionURL>
+
+
+<link rel="stylesheet" href="<%=request.getContextPath()%>/css/sw360.css">
+<link rel="stylesheet" href="<%=request.getContextPath()%>/webjars/jquery-ui/themes/base/jquery-ui.min.css">
+<link rel="stylesheet" href="<%=request.getContextPath()%>/webjars/jquery-confirm2/dist/jquery-confirm.min.css">
+<script src="<%=request.getContextPath()%>/webjars/jquery/dist/jquery.min.js" type="text/javascript"></script>
```

Comment:
Could you work with require on this page as well?

Reply:
I missed this comment. Could you please elaborate? RequireJS?

Reply:
Yeah, exactly, RequireJS. We are using it since some time but did not migrate everything - and wanted to do that when touch the pages. Since this page is completely new it would be good to use require. There is a wiki article about it over here: https://github.com/eclipse/sw360/wiki/Dev-Using-RequireJS-for-javascript-modules
But it might be good to learn from the usage on other pages, e.g. the one you added as well: frontend/sw360-portlet/src/main/webapp/html/admin/todos/view.jsp

Reply:
@sebastian-su please have a look

Reply:
@thegrumpylion apparently no update, could you pls. check?